README.md

```diff
@@ -22,6 +22,19 @@ See the wiki for detailed documentation.
 * Running locally in docker https://github.com/broadinstitute/firecloud-develop
 
 ##Unit Testing in Docker with MySQL
+Ensure that docker is up to date and initialized.
+Spin up mysql locally:
+```
+docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
```

Comment:
I updated the readme to describe running with custom mysql port numbers to mitigate this problem.